Target-MVD-Mathias/Networking/Services/UserAPI.swift

```diff
@@ -74,6 +74,27 @@ class UserAPI {
       failure(error)
     }
   }
+  
+  class func signup(name: String, email: String, password: String, gender: String, success: @escaping (_ responseObject: [String: Any]) -> Void, failure: @escaping (_ error: Error) -> Void) {
+    let parameters = [
```

Comment:
There are 2 spaces indentation here and in other lines there are 4, please be consistent